README.md

```diff
@@ -12,6 +12,7 @@ Check out this **[PYTHON ROADMAP](https://medium.com/@python.learning/the-roadma
 4. [star pattern 2](https://github.com/pythonites/Every-Python-Code-Ever/blob/main/codes%20inside/star_pattern_2.py)  ◤
 5. [star pattern 3](https://github.com/pythonites/Every-Python-Code-Ever/blob/main/codes%20inside/star_pattern_3.py)  ◢
 6. [star pattern 4](https://github.com/pythonites/Every-Python-Code-Ever/blob/main/codes%20inside/star_pattern_4.py)  ◥
+7. [factors of a number](https://github.com/pythonites/Every-Python-Code-Ever/blob/main/codes%20inside/factors_of_a_number.py)
```

Comment:
Did you chech link before addig?
It's showing 404 error
Please make necessary changes